pkg/apis/resource/namedresources.go

```diff
@@ -92,8 +92,16 @@ type NamedResourcesRequest struct {
 
 // NamedResourcesFilter is used in ResourceFilterModel.
 type NamedResourcesFilter struct {
-	// Selector is a selector like the one in Request. It must be true for
-	// a resource instance to be suitable for a claim using the class.
+	// Selector is a CEL expression which must evaluate to true if a
+	// resource instance is suitable. The language is as defined in
+	// https://kubernetes.io/docs/reference/using-api/cel/
+	//
+	// In addition, for each type NamedResourcesin AttributeValue there is a map that
```

Comment:
"NamedResourcesin" ?

Reply:
/lgtm cancel

Will fix. How ironic that a PR that fixes typos introduces new ones...

Reply:
Fixed: https://github.com/kubernetes/kubernetes/compare/20988de8cca8f7608987907a13c97767e1421ccb..acfaf8c27590c78fe9900f8c7c1e99927392c20e